data/ka'het missions.txt

```diff
@@ -0,0 +1,220 @@
+			`	"Then," she continues, "about a hundred years ago, a scout fleet arrived in a system where some Ka'het were still alive, which attacked the Remnant almost instantly and before they could activate the cloak some EMPs had already disabled two Starlings. They couldn't recover soon enough to survive, and the only remaining ship could do nothing but flee here as soon as possible."`
+			choice
+				`	"Which technology did you recover from the crashed Ka'het?"`
+					goto tech
+				`	"Wait, why did you call them 'shells' and not ships?"`
+					goto shells
+				`	"Have the Ka'het ever attacked the Remnant planets?"`
+			`	"Actually, we never met any Ka'het in the Ember Waste at all. The 'shells' we found had similar capabilities to the quantum keystone, meaning they could pass through the wormholes, so we don't know for sure why they remain there. Perhaps they aren't aware of their skills, or maybe they lost them over time."`
+			choice
+				`	"Wait, why did you call them 'shells' and not ships?"`
+					goto shells
+			label tech
+			`	"Back then we found large numbers of their EMP torpedoes, most of which in almost pristine conditions; their internal mechanisms were also simple enough we were able to reproduce them perfectly. We didn't find enough of the other systems in the 'shells' to do the same, but their tech still have been our main inspiration point for centuries."`
+			choice
+				`	"Wait, why did you call them 'shells' and not ships?"`
+			label shells
+			`	"You see, <first>," she says, "the Ka'het aren't normal spaceships. The debris we found were artificial exoskeletons, built around huge living beings; it's unlikely a species that big could have developed that sort of technology, so we presume someone must have created it for them. This could have happened in the Graveyard, since it's the only place where we know them to still live in, but we have such a limited data about that section of space that we can't say for sure." As she finishes talking, one of the people Taely's was talking with earlier comes closer and shows her something on a touchpad. "I have to go now, Captain," she tells you. "Good luck."`
+				decline
+	on decline
+		log "Asked Taely about the Graveyard and the Ka'het. She told me the previous contacts between them and the Remnant."
+		log "Factions" "Ka'het" `The Ka'het are aliens which live in an area south of the Ember Waste, called Graveyard, and attack everyone on sight. They are large lifeforms which can travel in space using artificial exoskeletons.`
+
+mission "First Contact: Ka'het: Quarg 1A"
```

Comment:
How about instead of having a number of missions that account for the various Quarg that the player may or may not know (which would need to be expanded upon as new Quarg locations are added), how about you just have a single mission that tells you to find a Quarg to speak to, and the player can then land on any Quarg location to get their side of the story, then that mission speaking with the Quarg fails this one. It would be far simpler than what you're trying to do here. (Think of it like when you first contact the Wanderers after also having visited the Hai. That mission doesn't point you directly to Hai-home, it just tells you in general to look for a Hai to help you.)

Reply:
But are you sure that would solve the issue I currently have of the mission failing the *next* landing instead of when you do the second mission? In the meanwhile, I will change it to directly point the player to Lagrange (e.g. closest quarg station) and directly remove the rest